src/codegen/detail/code_gen_c.h

```diff
@@ -47,38 +47,47 @@ template <class Stream> void CodeGenC<Stream>::visit(const VarDef &op) {
 
     if (op->buffer_->atype() == AccessType::Cache) {
         // e.g. float x[5][5][5];
```

Comment:
Stream manipulation is hard to read. Better write what will be output in a comment like this.